src/main/java/org/jabref/gui/desktop/os/NativeDesktop.java

```diff
@@ -31,6 +30,19 @@ public interface NativeDesktop {
      */
     Path getApplicationDirectory();
 
+    /**
+     * Get the user's default file chooser directory
+     *
+     * @return The path to the directory
+     */
+    default Path getDefaultFileChooserDirectory() {
+        // We do not return a subdirectory "JabRef", because
```

Comment:
isn't it better to enforce that this method is properly implemented on each os? (similar to the application dir above)

Reply:
We have it implemented for LInux/Mac/Windows

Reply:
so why is a default implementation needed?

Reply:
Default method is an artifact from development. Can probably reduced to the plain method declaration, i think.

Reply:
I removed it